packages/create-llama/helpers/typescript.ts

```diff
@@ -98,6 +99,27 @@ export const installTSTemplate = async ({
     );
   }
 
+  if (observability === "opentelemetry") {
+    const nextConfigJsonFile = path.join(root, "next.config.json");
+    const nextConfigJson: any = JSON.parse(
+      await fs.readFile(nextConfigJsonFile, "utf8"),
+    );
+
+    nextConfigJson.webpack.modules = {
```

Comment:
integrate condition from above:
```suggestion
  if (framework === "nextjs") 
    const nextConfigJsonFile = path.join(root, "next.config.json");
    const nextConfigJson: any = JSON.parse(
      await fs.readFile(nextConfigJsonFile, "utf8"),
    );

     if (!backend) {
         nextConfigJson.output = "export";
         nextConfigJson.images = { unoptimized: true };
     }
     if (observability === "opentelemetry") {
    nextConfigJson.webpack.modules = {
```